### PR TITLE
Use libmozdata versions stuff and avoid to nag when inconsistencies i…

### PR DESCRIPTION
**Pull request by calixteman**
…n versions (p-d vs bugzilla)

### CODE REVIEW

File: auto_nag/scripts/tracked_needinfo.py [hunk 1]
```diff
@@ -11,7 +11,10 @@ class TrackedNeedinfo(BzCleaner, Nag):
     def __init__(self, channel):
         super(TrackedNeedinfo, self).__init__()
         self.channel = channel
-        self.versions = utils.get_checked_versions()
+        if not self.init_versions():
+            self.version = '??'
```
Comment (marco-c) [inline auto_nag/scripts/tracked_needinfo.py]: Why?

File: auto_nag/utils.py [hunk 2]
```diff
@@ -495,46 +505,17 @@ def get_human_lag(date):
...
-    }
-    return _CURRENT_VERSIONS
-
+def get_nightly_version_from_bz():
+    def bug_handler(bug, data):
+        status = 'cf_status_firefox'
+        N = len(status)
+        for k in bug.keys():
+            if k.startswith(status):
+                k = k[N:]
```
Comment (marco-c) [inline auto_nag/utils.py]: Nit: remove N variable and just do `k = k[len(status):]`

File: auto_nag/utils.py [hunk 3]
```diff
@@ -495,46 +505,17 @@ def get_human_lag(date):
...
-def get_versions(channel=None):
-    versions = get_current_versions()
-    if channel and isinstance(channel, six.string_types):
-        channel = channel.lower()
-        if channel == 'nightly':
-            channel = 'central'
-        if channel in versions:
-            return versions[channel]
+    data = []
+    Bugzilla(bugids=['1234567'], bughandler=bug_handler, bugdata=data).get_data().wait()
```
Comment (marco-c) [inline auto_nag/utils.py]: There is an API call to get the list of all Bugzilla fields, maybe we could use that instead.
Not really a big difference though.

File: auto_nag/bzcleaner.py [hunk 4]
```diff
@@ -25,6 +25,7 @@ def __init__(self):
         self.has_flags = False
         self.cache = Cache(self.name(), self.max_days_in_cache())
         self.test_mode = utils.get_config('common', 'test', False)
+        self.versions = 'nothing'
```
Comment (marco-c) [inline auto_nag/bzcleaner.py]: Couldn't you use None as the default instead?

File: auto_nag/utils.py (hunk 2 above)
Comment (calixteman) [reply auto_nag/utils.py]: yeah, it's just to avoid a conflict between black and flake8

File: auto_nag/utils.py (hunk 3 above)
Comment (calixteman) [reply auto_nag/utils.py]: That would require to add this api endpoint in lmd.
I'm fine that so file a bug on lmd and I'll add the feature.

**Pull request merged**